internal/shipment/repository.go

```diff
@@ -0,0 +1,74 @@
+package shipment
+
+import (
+	"gorm.io/gorm"
+)
+
+type APIShipmentRepo interface {
+	Create(s Shipment) (error, string, int)
+	FindAll() ([]Shipment, error, string, int)
+	FindOne(id int) (Shipment, error, string, int)
+	Update(s Shipment) (error, string, int)
+	Delete(id int) (error, string, int)
```

Comment:
not a big fun of return http status code in repo layer, code show attach to error

Reply:
please add an index on userId